pkg/server/chaosd/stress_attack.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package chaosd
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/chaos-mesh/chaos-mesh/pkg/bpm"
+	"github.com/google/uuid"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/shirou/gopsutil/process"
+	"go.uber.org/zap"
+
+	"github.com/chaos-mesh/chaosd/pkg/core"
+)
+
+const (
+	StressAttack = "stress attack"
+)
+
+func (s *Server) StressAttack(attack *core.StressCommand) (string, error) {
+	var err error
+	uid := uuid.New().String()
+
+	if err := s.exp.Set(context.Background(), &core.Experiment{
+		Uid:            uid,
+		Status:         core.Created,
+		Kind:           StressAttack,
+		RecoverCommand: attack.String(),
+	}); err != nil {
+		return "", errors.WithStack(err)
+	}
+
+	stressngPid := ""
+
+	defer func() {
+		if err != nil {
+			if err := s.exp.Update(context.Background(), uid, core.Error, err.Error(), attack.String()); err != nil {
+				log.Error("failed to update experiment", zap.Error(err))
+			}
+			return
+		}
+
+		// use the stressngPid as recover command, and will kill the pid when recover
```

Comment:
Why not adding a `stressngPid` on stressCommand struct? Because users also want to know the configuration of this experiment when they search for this experiment.

Reply:
good catch, updated